src/features/user/MFV2/CommunityContributions/communityContributions.module.scss

```diff
@@ -141,15 +136,17 @@
   display: flex;
   flex-direction: column;
   gap: 12px;
-  font-size: 0.75em;
+  font-size: $fontSmall;
```

Comment:
Is the size deliberately increased here since according to `0.75em` it should be `$fontXSmall`

Reply:
Yes, tried to make font sizes consistent across the screen, so I used the predefined sizes.

Reply:
`fontSmall` is `14px` though